jans-bom/pom.xml

```diff
@@ -13,19 +13,47 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.min-version>3.0.3</maven.min-version>
 
-		<resteasy.version>4.5.7.Final</resteasy.version>
+		<resteasy.version>6.0.0.Final</resteasy.version>
 
 		<httpcore.version>4.4.14</httpcore.version>
 		<httpclient.version>4.5.13</httpclient.version>
 
-		<weld.version>3.1.8.Final</weld.version>
-		<servlet.version>5.0.0</servlet.version>
+		<weld.version>4.0.3.Final</weld.version>
 		<richfaces.version>4.5.19-gluu.Final</richfaces.version>
-
-		<jsf-api.version>2.2.20</jsf-api.version>
-		<jsf-impl.version>2.3.14</jsf-impl.version>
-		<uel-api.version>2.2.1-b04</uel-api.version>
-		<uel-impl.version>2.2.1-b05</uel-impl.version>
+		
+		<!-- Jakarta -->
+		<jakarta.jakartaee.api.version>9.0.0</jakarta.jakartaee.api.version>
```

Comment:
This section is over-managed. The main part here which specify compatible versions is Weld. Hence we import weld-core-parent [pom](https://mvnrepository.com/artifact/org.jboss.weld/weld-core-parent/4.0.3.Final) file. It should provides information about compatible versions. If we need to override specific libs we need to add comments to this definition to explain the reason of this override. Now we can get into similar to DLL hell situation.